backend/src/main/scala/cromwell/backend/BackendInitializationData.scala

```diff
@@ -3,6 +3,23 @@ package cromwell.backend
 // single-backend "marker interface"
 trait BackendInitializationData
 
+object BackendInitializationData {
+
+  /**
+    * Utility for extracting a type of backend data that we know is wrapped in an Option.
+    * @param initializationDataOption The "optional" backend data passed usually from the initialization actor to each
+    *                                 execution actor.
```

Comment:
ToL: I don't know about _\<insert future backend here\>_, I think most backends should be caching some computed values, then passing it into execution actors. In the case of the SFS this includes things like the GCS file system used for localization.